src/overlays/actors/ovl_En_Ge3/z_en_ge3.h

```diff
@@ -2,14 +2,54 @@
 #define Z_EN_GE3_H
 
 #include "global.h"
+#include "z64snap.h"
+#include "objects/object_geldb/object_geldb.h"
 
 struct EnGe3;
 
 typedef void (*EnGe3ActionFunc)(struct EnGe3*, PlayState*);
 
+#define GERUDO_AVEIL_GET_PATH(thisx) (((thisx)->params & 0xFC00) >> 10)
+#define GERUDO_AVEIL_GET_EXIT(thisx) (((thisx)->params & 0x1F0) >> 4)
+#define GERUDO_AVEIL_GET_TYPE(thisx) ((thisx)->params & 1)
+
+#define GERUDO_AVEIL_PATH_NONE 0x3F
```

Comment:
These seem to be more general than specific to actors (most use 0x3F I think, though there are a few outliers here and there).

Reply:
Yeah, we'll have to do a general review of params etc. once all the actors are done, but at least having them with defines like this will make it easier to work on later.

Reply:
Doesnt help that sometimes that value is 1F or 7F either

Reply:
Good amount of work since this thread. I think `GERUDO_AVEIL_PATH_INDEX_NONE` would be appropriate now?